pgmax/fg/graph.py

```diff
@@ -121,6 +123,7 @@ def add_factor(
                 If None, it is assumed the log potential is uniform 0 and such an array is automatically
                 initialized.
         """
+        # TODO: change
```

Comment:
We should indeed support adding individual OR factors. One possibility is to change this function. Another is to have a new function which takes a factory function and other arguments, similar to the current `add_factor_group` function, just for adding different kinds of specialized factors.